packages/app/package.json

```diff
@@ -139,6 +140,7 @@
     "hash-sum": "^1.0.2",
     "humps": "CompuIves/humps",
     "immer": "^0.8.5",
+    "immutability-helper": "^2.6.6",
```

Comment:
(Possibly) not required. I've used it manually update the state, as calling `setState` multiple times in a row, to push to an array can